### PR TITLE
Convert double to integer via truncation.

### LINKED ISSUE #191: Standardize double-to-integer conversion via truncation
Tests were added for double-to-integer conversion specifying round-half-away semantics (following GoogleSQL), but the implementations used the more common truncation (round towards zero), as is done natively in C, Go, and Java.

We'll standardize on round-towards-zero to match the implementations and the more common expectation of programmers.

### PR DESCRIPTION
**Pull request by JimLarson**
Matches behavior of C, Go, and Java.

Introduce a round() function for round-half-away like GoogleSQL.

Fixes #191

### CODE REVIEW

File: doc/langdef.md [hunk 1]
```diff
@@ -2070,7 +2070,8 @@ See [cel-go/issues/9](https://github.com/google/cel-go/issues/9).
       (double) -> int
     </td>
     <td>
-      type conversion
+      Type conversion. Rounds toward zero, then errors if result is out of
+      range. See also round().
```
Comment (TristonianJones) [inline doc/langdef.md:2074]: Let's introduce `round`, `floor`, and `ceil` into a `math` namespace in a separate PR. 
e.g. `math.round(double)`, etc.

Comment (JimLarson) [reply doc/langdef.md:2074]: Done.

**Comment by JimLarson** [comment]
PTAL

**Comment by JimLarson** [comment]
Filed Issue #193 as a reminder to implement the math extension library with alternate ways of rounding.

**Review by TristonianJones — APPROVED** [review]

**Pull request merged**